draft-specifications/CapTP Specification.md

```diff
@@ -258,25 +258,23 @@ public keys and the SHA512 hash algorithm. The EdDSA public keys are based on
 the Ed25519 elliptic curve. The public key is formatted as follows:
 
 ```text
-(public-key (ecc (curve Ed25519) (flags eddsa) (q ...) (s ...)))
+['public-key ['ecc ['curve 'Ed25519] ['flags 'eddsa] ['q q_value]]]
```

Comment:
Our spec’s “presentation format” (what I was calling “abstract notation”) doesn’t yet have an explicit affordance for interpolated variables like `q_value`, but as far as I’m concerned, this is fine. We can back-fill bare identifiers, as long as they are not possible to confuse for a shorthand notation on symbols.